### DISCUSSION TITLE
AMD gpu

### DISCUSSION THREAD

**Discussion by Fahad021** [dask/dask#7549] [Q&A]
Newbie here. Is dusk AMD-gpu friendly? For example- my laptop comes with AMD Radeon (TM) Pro WX 3200 Graphics card.

**Comment by jrbourbeau** [comment]
It appears CuPy has experimental support for AMD (https://docs.cupy.dev/en/stable/install_rocm.html) so Dask array _might_ work. Though perhaps someone on @dask/gpu can give a more detailed answer

**Reply by kkraus14** [reply]
10 of the 11 members of the dask/gpu group (myself included) are NVIDIA employees so I think we'll refrain from answering this question 😆

**Reply by quasiben** [reply]
It's possible things might just work as @jrbourbeau said but it's untested.  I would suggest starting with Dask running through a handful of exercises.  Transpose-sum (below) or [svd](https://examples.dask.org/machine-learning/svd.html) would be good starting points:

```python
rs = dask.array.random.RandomState(RandomState=cupy.random.RandomState)
a = rs.random((10000, 10000), chunks=100)
out = a + a.T
```

As you get to multiple workers, serialization might a problem though Numba does have some support for ROCm as well.  Reiterating @kkraus14's point, those of us at NVIDIA have limited time/resources/etc to explore here
